summit/index.html

```diff
@@ -511,17 +424,19 @@ <h4>Help! I’m still confused! Where can I ask more questions?</h4>
 
 		<footer>
 			<img alt="Footer" src="../assets/footer.svg" />
-			<p>
-				Questions, Comments, Concerns?
-				<br/>
-				Reach out to <a target="_blank" href="mailto:hello@impactlabs.io">hello@impactlabs.io</a>
-				<br/>
-				<br/>
-				Move purposefully and fix things. 
-				<br/>
-				<br/>
-				Copyright &copy; Impact Labs 2019. <a target="_blank" href="https://docs.google.com/document/d/11rG_I1-jbV9INWTXWyhvDNLHPxTPtrBW7ZMhqVfc2b8/edit">Code of Conduct</a>
-			</p>
+			<div class="content">
```

Comment:
You need to make this change in the Fellowship, Coalition, and Team page as well - see below:
![image](https://user-images.githubusercontent.com/17170744/77240092-599f7b80-6bb8-11ea-91e9-f8d71f347ee0.png)